engine/src/main/java/org/terasology/world/viewer/layers/NominalFacetLayer.java

```diff
@@ -56,8 +56,10 @@ public void render(BufferedImage img, Region region) {
         for (int z = 0; z < height; z++) {
             for (int x = 0; x < width; x++) {
                 Color src = getColor(facet, x, z);
-                int blend = blender.get(src.rgba());
-                dataBuffer.setElem(z * width + x, blend);
+                if (src != null) {
```

Comment:
If an external module of unknown quality does not play be the rules, it causes chaos.  I agree that it *should* never be null.   I found this when getting shatteredplanes to map to the Core biomes